packages/aragon-wrapper/src/rpc/handlers/get-forwarded-actions.js

```diff
@@ -0,0 +1,22 @@
+import { map, filter } from 'rxjs/operators'
+
+export default function (request, proxy, wrapper) {
+  // filter out actions that aren't targeted
+  // at the subscribed app proxy
+  const getProxyActions = actionsArray => (
+    actionsArray
+    .filter(action => (action.target === proxy.address))
+  )
+
+  return wrapper.forwardedActions.pipe(
+    // transform the observable into an event-like object
+    // that only contains actions targeting the proxy
+    map(actions => ({
+      event: 'ForwardedActions', 
```

Comment:
I think the structure we want might be event: { eventName: 'ForwardedActions', returnValues: getProxyActions(actions) }

Reply:
we alias `event` to be `eventName` within our script.js files: https://github.com/AutarkLabs/planning-suite/blob/6927c8fb9d250218c096b720fc52e509a118ad2c/apps/rewards/app/store/events.js#L7